case_studies/dc2/dc2_shear_conv.ipynb

```diff
@@ -0,0 +1,764 @@
+{
```

Comment:
We should keep this notebook in a separate case study because @YicunDuanUMich is working on this one. Perhaps the current DC2 case study could be renamed something like `dc2_cataloging`.